ts/src/config.ts

```diff
@@ -106,6 +106,14 @@ export interface Config extends AuthenticationConfig {
   // Allows user to specify API URL other than
   // https://cloudprofiler.googleapis.com/v2.
   baseApiUrl?: string;
+
+  // Time to pause between profile collection.
+  // For testing with startLocal() only.
+  profileCollectionPauseMillis?: number;
+
+  // Duration of time profiles collected when using startLocal().
+  // For testing with startLocal() only.
+  timeDurationMillis?: number;
```

Comment:
Similar - something like "localTimeDurationMillis". "local time" reads a bit weird, like it's time in local time zone, but oh well. Suggest a better name if you can think of something.